### ISSUE TITLE
cmd results --details -o json does not show all reports

### ISSUE THREAD

**Issue by dkrupp** [Ericsson/codechecker#2408]
This will show only bugs with review status either 1 or 0 (status 2 is consistently not shown):
•	CodeChecker cmd results --details --url localhost:8001/Default "{run}" -o json

This should show reports from all status.

This will show bugs with any review status 0, 1, 2 (didn't try others):
•	CodeChecker cmd results --details --url localhost:8001/Default "{run}" --review-status -o json
--review-status CONFIRMED works as expected and displays only confirmed bugs.

**Comment by csordasmarton** [comment]
@dkrupp The default review status filter values of the CodeChecker cmd results are `unreviewed` and `confirmed`.
 [See](https://github.com/Ericsson/codechecker/blob/master/docs/web/user_guide.md#cmd) the help of this command:
```sh
--review-status [REVIEW_STATUS [REVIEW_STATUS ...]]
                        Filter results by review statuses. (default:
                        ['unreviewed', 'confirmed'])
```
The default filter values on the UI are the same, so what is the reason we should change this behavior.

**Comment by dkrupp** [comment]
Sorry it is fine like this. I remember that we do not show false positives by default.  I will close the ticket.
